examples/question_answering_model_confidence.py

```diff
@@ -0,0 +1,111 @@
+# fmt: off
+import logging
+from pathlib import Path
+
+from farm.data_handler.data_silo import DataSilo
+from farm.data_handler.processor import SquadProcessor
+from farm.modeling.adaptive_model import AdaptiveModel
+from farm.modeling.tokenization import Tokenizer
+from farm.eval import Evaluator
+from farm.utils import set_all_seeds, initialize_device_settings
+
+def question_answering_model_confidence():
+    logging.basicConfig(
+        format="%(asctime)s - %(levelname)s - %(name)s -   %(message)s",
+        datefmt="%m/%d/%Y %H:%M:%S",
+        level=logging.INFO,
+    )
+
+    ##########################
+    ########## Settings
+    ##########################
+    set_all_seeds(seed=42)
```

Comment:
Do we need randomized seed here or user supplied?

Reply:
I think Julian just took standard code from other examples and I believe the seed is rather irrelevant here.

Reply:
oh okay thanks for clarification. I don't have much idea thats why asked.